### PR TITLE
Single tab refactoring

### PR DESCRIPTION
**Pull request by spacewoox**
-The single tab has been renamed to 'vdi' and is now a route.
-Non-reusable components have been removed.
-vdi template has been splitted into partial for better readability
-vdi controller now injects multiple mixins for separation of concerns.

### CODE REVIEW

File: assets/app/vdi/load/template.hbs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{{#unless showVdi}}
+  <div class="vdi-load-background centered fixed noselect">
+    <div>
+      <h2 class='text'>Loading</h2>
+      <div class='white '>
+        Please wait, everything is gonna be fine.
```
Comment (dynamiccast) [inline assets/app/vdi/load/template.hbs]: We should think of something less familiar

**Comment by dynamiccast** [comment]
The file explorer to publish an application never shows files. Acutally no XHR requests are ever made to the backend.

**Comment by dynamiccast** [comment]
Cancelling the image create never cancel the pending request to POST /api/images.

**Comment by dynamiccast** [comment]
The following warning appears in the console when I create an image:

```
A property of <nanocloud@view:-outlet::ember545> was modified inside the didInsertElement hook. You should never change properties on components, services or models during didInsertElement because it causes significant performance degradation. [deprecation id: ember-views.dispatching-modify-property]
```

**Comment by corentindrouet** [comment]
This pr fixe #274

**Comment by dynamiccast** [comment]
Apps publication still does not work.
The file listing do not query the API properly. `StorageController::file` line 183 should be modify to accept any value for machine and not only `"true"`.

**Comment by dynamiccast** [comment]
CTRL + F to find something in the file browser to publish on app if grabbed by Guacamole.

**Comment by dynamiccast** [comment]
Not able to write anything in the clipboard area.

**Pull request merged**